docs/docs/cmd/spo/list/list-roleinheritance-reset.md

```diff
@@ -11,26 +11,35 @@ m365 spo list roleinheritance reset [options]
 ## Options
 
 `-u, --webUrl <webUrl>`
-: URL of the site where the list is located
+: URL of the site.
```

Comment:
Is there a reason why we're shortening this description? The original was pretty descriptive